data/vala-linter-1.0.pc.in

```diff
@@ -0,0 +1,13 @@
+prefix=@prefix@
+exec_prefix=${prefix}
+libdir=@libdir@
+datadir=${prefix}/share
+includedir=${prefix}/include
+
+Name: vala-linter-1.0
+Description: Lqibrary prividing a Vala code files checker for code-style errors
```

Comment:
Some typos here.

Reply:
First two lines there are not typos. but the line at Description, fixed.

Reply:
Version of libvala will be set to `>= 0.40.4` because is the version available in Ubuntu 18.04, so elementary 5.x